bambou/session.go

```diff
@@ -279,7 +311,7 @@ func (s *Session) FetchEntity(object Identifiable) *Error {
 
 	request, err := http.NewRequest("GET", url, nil)
 	if err != nil {
-		return NewError(http.StatusBadRequest, err.Error())
+		return NewBambouError("", err.Error())
```

Comment:
Why are we not returning the bad request code as title? Not a big fan of empty titles